roles-and-groups.md

```diff
@@ -0,0 +1,152 @@
+  - **Get noticed**: Consistency and valuable contributions will create more project involvement and reliability.
+  - **Review eligibility**: A minimum number of approved PRs and a reasonable time contributing to the project (3-6 months) is the general acceptable baseline for eligibility.
+
+### Committers
+
+A Committer has specific rights for the project.
+
+As an example a Committer can assign people to issues, merge PRs, or create branches directly in a repository of the project.
+Any Committer or Maintainer of the given project should nominate a Contributor to become a Committer based on the project's needs and following these steps:
+
+- **Define your project's needs**: A repository should have minimum 3 committers to ensure a project's sustainability and resilience. It will also allow for fairness in the votes. Project owners can choose to have more committers depending on the size of their project.
+- **Evaluate your project's performance**: Committers and Maintainers should monitor their project health and velocity and make decisions about whether the project needs more Committers.
+- **Verify your project's needs**: Committers or Maintainers should define the number of good contributions that a Contributor must reach to consider a nomination. This is based on the size and complexity of the repo in question.
+- **Nominate more resources**: To promote a contributor, a vote should be held in a GitHub PR (by doing a change to the config.yaml file) and by making mention of the basis reached for nominating the Contributor.
+- **Approve the nomination**: All Maintainers should vote on that nomination and the vote passes when reaching majority of votes.
+- **Make adjustments**: Maintainers should be responsible for removing inactive committers. A committer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+As long as a Committer follows our Code of Conduct, makes reasonable contributions to the project,
+and is not violating any of our policies, they will keep their Committer role.
+
+Project maintainers may remove a committer if:
+
+- Breaks Code of Conduct.
+- Shown no activity for 6 months and does not respond when contacted by the project maintainers or LFDT staff.
+- Keeps making bad decisions (their approvals and contributions are actively affecting the project's performance).
+
+The addition or removal of committers must be decided by a project maintainer vote. The voting must be held via a GitHub PR (by doing a change to the `config.yaml` file).
+
+### Maintainers
+
+The Maintainers of the project are the people that are responsible for the project.
+
+A Maintainer should drive the project's roadmap/vision and take care of topics like community meetings,
+the general structure of the project (fits our best practices, is secure, ...), and so on.
+
+In addition, the Maintainers should report project updates to the Hiero **Technical Steering Committee** (TSC).
```

Comment:
Do maintainers present in the TSC meeting or how should updates be provided?

Reply:
TSC meeting by adding it to the agenda, i can add that as additional information.